### DISCUSSION TITLE
Aditional MAX7219CWG-2130

### DISCUSSION THREAD

**Discussion by Berghausa** [shanteacontrols/OpenDeck#188] [Q&A]
Heyy,  just asking if it possible adding an addional Led Driver?
If yes, does the Configurator accept it as Driver or do i need to change the code to?

Im trying to make a single pcp with the switches and sliders solderd to the pcp and i want to add more led for the buttons.
Main use would be for Voicemeeter Potato.

**Comment by paradajz** [comment]
It's hardcoded to just 1 at the moment, but it could be expanded. You would just need to write your own YAML config and then load that to the board in question.

**Reply by paradajz** [reply]
FYI this is now possible.
